lerobot/configs/policy/florence_policy.yaml

```diff
@@ -0,0 +1,77 @@
+# @package _global_
```

Comment:
rename `florence.yaml` for consistency with other policies

Reply:
Get it! I have renamed all "florence_policy" to "florence"